cabal.project

```diff
@@ -5,6 +5,10 @@ constraints: unix >= 2.7.1.0
 -- Uncomment to allow picking up extra local unpacked deps:
 --optional-packages: */
 
+-- Remove after hackage-repo-tool release
```

Comment:
If this works, I suggest we just make a Hackage revision for `hackage-repo-tool` instead of updating `appveyor.yml` as well.

Reply:
appveyor doesn't use hackage-repo-tool yet :)

Reply:
Oh right. Anyway, I just checked that `cabal new-build --allow-newer=optparse-applicative --constraint="optparse-applicative == 0.14.0.0"` works with `hackage-repo-tool-0.1.1`, so I asked @hvr to bump that constraint to `< 0.15`.

Reply:
Apparently he wants to wait until a new release, which will also include the Windows patch.